pom.xml

```diff
@@ -199,7 +199,7 @@
                         <Implementation-Version>${spec.implementation.version}</Implementation-Version>
                         <Specification-Version>${spec.specification.version}</Specification-Version>
                         <Bundle-Description>
-                            JACC API ${spec.version} Design Specification
+                            Jakarta Authorization ${spec.version} Design Specification
```

Comment:
"Design" doesn't need to be part of the specification name.

Reply:
Agreed, I removed it.